x-pack/packages/kbn-entities-schema/src/schema/entity_definition.ts

```diff
@@ -59,4 +59,20 @@ export const entityDefinitionSchema = z.object({
   installStartedAt: z.optional(z.string()),
 });
 
+export const entityDefinitionUpdateSchema = entityDefinitionSchema
+  .omit({
+    id: true,
+    managed: true,
+    installStatus: true,
+    installStartedAt: true,
+  })
+  .partial()
+  .merge(
+    z.object({
+      history: z.optional(entityDefinitionSchema.shape.history.partial()),
+      version: semVerSchema,
+    })
+  );
+
 export type EntityDefinition = z.infer<typeof entityDefinitionSchema>;
+export type EntityDefinitionUpdate = z.infer<typeof entityDefinitionUpdateSchema>;
```

Comment:
I wondering if there is a better name than Update, maybe Patch or Delta, something to express that it's a partial thing that will be applied?